labgui.egg-info/PKG-INFO

```diff
@@ -1,69 +0,0 @@
-Metadata-Version: 1.1
```

Comment:
Any specific reason you deleted those files? I think there were automatically created by cookiecutter

Reply:
I don't believe it should be in the public repository, as it's sole purpose is for distribution. I added it to .gitignore for the same reason build/ and dist/ are in it.

A possible solution would be to have it available on another stream, such as develop, release, or possibly even a new one meant solely for distribution

Reply:
You're right, I'll re-add them on a specific branch later.